workshops/forms.py

```diff
@@ -43,10 +49,11 @@ class InstructorsForm(forms.Form):
                                  min_value=-180.0,
                                  max_value=180.0,
                                  required=False)
-    airport = forms.ModelChoiceField(label='airport',
-                                     queryset=Airport.objects.all(),
-                                     to_field_name='iata',
-                                     required=False)
+    airport = selectable.AutoCompleteSelectField(
+                lookup_class=lookups.AirportLookup,
+                label='Airport',
+                required=False,
+                help_text=AUTOCOMPLETE_HELP_TEXT)
```

Comment:
@pbanaszkiewicz We lost the conversation here because of the interactive rebase session, but you can see that the only-line changed is the `airport` line, so the blame should be intact as you originally requested.